packages/foreman/foreman-proxy/foreman-proxy.spec

```diff
@@ -28,6 +28,9 @@ Requires:       ruby(release) >= 2.5
 Requires:       rubygems
 Requires:       rubygem(bundler_ext)
 
+# Require fapolicyd package if fapoolicyd is present
+Requires: (%{name}-fapolicyd if fapolicyd)
```

Comment:
when this is in place and we've sufficiently tested the flow, I think we can drop the `foreman-fapolicyd → foreman-proxy-fapolicyd` require that you added, as the installation would now be "managed" by the packages here

Reply:
https://github.com/theforeman/foreman-packaging/pull/9901